<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Expanses</class>
 <widget class="QDialog" name="Edit_Expanses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="4">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="editButton">
      <property name="text">
       <string>Редактировать</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="delButton">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Отдел</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>Edit_Expanses</receiver>
   <slot>updateTable(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>Edit_Expanses</receiver>
   <slot>addExpense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editButton</sender>
   <signal>clicked()</signal>
   <receiver>Edit_Expanses</receiver>
   <slot>editExpense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delButton</sender>
   <signal>clicked()</signal>
   <receiver>Edit_Expanses</receiver>
   <slot>deleteExpense()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateTable(int)</slot>
  <slot>addExpense()</slot>
  <slot>deleteExpense()</slot>
  <slot>editExpense()</slot>
 </slots>
</ui>
